shesha-reactjs/src/components/dataTable/index.tsx

```
@@ -114,6 +117,17 @@ export const DataTable: FC<Partial<IIndexTableProps>> = ({
   const [visibleColumns, setVisibleColumns] = useState<number>(0);
   const appContextData = useApplicationContextData();
 
+  // Safely get message API - may not be available in all contexts
+  let message = null;
+  try {
+    const app = App.useApp();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)